### DISCUSSION TITLE
How I can get MapEventProps?

### DISCUSSION THREAD

**Discussion by Mifaresss** [visgl/react-google-maps#157] [Q&A]
I mean this:
![image](https://github.com/visgl/react-google-maps/assets/106471161/c6a32085-3abe-4d26-8ffd-6134e567dc62)

**Comment by usefulthink** [comment]
They are merged into the exported `MapProps` type:

https://github.com/visgl/react-google-maps/blob/eb3fb7dc2efd0f797de87cce604952b3c051491e/src/components/map/index.tsx#L31-L60

Do you need them as an individual exported type? What for?

**Reply by Mifaresss** [reply]
Yes, I need individual type. For handlers. For example:
`const onClickMapHandler = (event: MapMouseEvent) => {
 // some logic with event
}`

**Reply by usefulthink** [reply]
Oh yes that absolutely makes sense. I was expecting that most of the time the type-inference would be enough, but in a case like that definitely isn't. Can you create a feature-request for me to export the event-types in case I forget?

**Reply by Mifaresss** [reply]
Ok, I do)

**Comment by usefulthink** [comment]
should be available in v0.5.0 later today: https://github.com/visgl/react-google-maps/pull/167
